ext/standard/filestat.c

```diff
@@ -911,7 +905,19 @@ PHPAPI void php_stat(const char *filename, size_t filename_length, int type, zva
 		RETURN_TRUE; /* the false case was done earlier */
 	case FS_LSTAT:
 		/* FALLTHROUGH */
-	case FS_STAT:
+	case FS_STAT: {
```

Comment:
If I need to declare new variables inside a `case`, I need to open a new scope, right?

Reply:
C99 (which master adheres to) allows to mix declarations and statement, so unless you want to keep the variables local to the case, you don't *need* a block.